plugins/theme-deuteranopia/src/index.ts

```diff
@@ -11,79 +11,79 @@
 import * as Blockly from 'blockly/core';
 
 const defaultBlockStyles = {
-  colour_blocks: {
+  colourBlocks: {
```

Comment:
So, renaming these properties will break the theme.  This is one case where you have _suppress_ the eslint errors, rather than changing the code so there isn't an error:

```suggestion
  colour_blocks: { // eslint-disable-line camelcase
```

(Here and many other places in this PR.)

Reply:
It doesn't seems to destroy the theme for me after my tests

Reply:
So tell me if you want me to do it but the one or the other isn't breaking so...

Reply:
Please revert the property name changes.  The theme won't work if your rename them.

Reply:
Please apply the suggested change above, and equivalent changes to each of the other names below.